app/assets/src/components/views/samples/SamplesView.jsx

```diff
@@ -204,13 +208,18 @@ class SamplesView extends React.Component {
   renderHeatmapTrigger = () => {
     const { selectedSampleIds } = this.state;
     return (
-      <a
-        href={`/visualizations/heatmap?sampleIds=${Array.from(
-          selectedSampleIds
-        )}`}
-      >
-        <HeatmapIcon className={cs.icon} />
-      </a>
+      <BasicPopup
```

Comment:
Should we add this popups inside the icons?

Reply:
sure. looks like the icons were only used here, so I moved them in